hydra-remote_identifier.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activesupport', '>= 3.2.13'
   spec.add_dependency 'rest-client', '~> 2.0.2'
 
-  spec.add_development_dependency "bundler", "~> 1.3"
+  spec.add_development_dependency "bundler"
```

Comment:
Is there a reason we relaxed this minimum version restriction?  The `~> 1.3` was probably put there for a good reason.

Reply:
Added this back in, in the future if anyone tries to bundle they'll probably have 2 or greater, so they'll need to use `bundle _1.3_ install` to bundle specifically with version 1.3